Assignment2/jorge5799-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,59 @@
+{
+    "@context": {
+        "class": "http://www.weather.net/Class#",
+        "sensor": "http://www.weather.net/Sensor#",
+        "measurement": "http://www.weather.net/Measurement#",
+        "computer": "http://www.weather.net/Computer#",
+        "user": "http://www.weather.net/User#",
+        "properties": "http://www.weather.net/Properties#",
+        "Class01": "class:Class01",
+        "Sensor029": "sensor:Sensor029",
+        "Computer101": "computer:Computer101",
+        "User10A": "user:User10A",
+        "hasMeasurements": "properties:hasMeasurements",
+        "hasTemperature": "properties:hasTemperature",
+        "atTime": "properties:atTime",
+        "hasOwner": "properties:hasOwner",
+        "hasName": "properties:hasName"
+    },
+    "Class01": {
+        "@id": "http://www.weather.net/Class01",
+        "@type": "class:Class01",
+        "includes": [
+            {
+                "Sensor029": {
```

Comment:
You don't need to define "Sensor029": because it isn't an attribute, it is a resource, thus I suggest you remove it. The Sensor09's content is ok, I mean leave the attributes you defined inside its curly braces

This comment also applies for "Computer101":, "Measurement8401":, and "User10A":. You don't need to define them, only leave its content.

Reply:
Please, remove "Sensor029":, "Computer101":, "Measurement8401":, and "User10A":